rest/rest-resources/src/main/java/org/apache/brooklyn/rest/util/MultiSessionAttributeAdapter.java

```diff
@@ -638,12 +638,22 @@ public MultiSessionAttributeAdapter resetExpiration() {
         Handler[] hh = getSessionHandlers();
         if (hh!=null) {
             for (Handler h: hh) {
-                Session ss = ((SessionHandler)h).getSession(getId());
-                if (ss!=null) {
+                Session ss = getSessionSafely(h, getId());
+                if (ss != null) {
                     ss.setMaxInactiveInterval(maxInativeInterval);
                 }
             }
         }
         return this;
     }
+
+    private static Session getSessionSafely(Handler h, String id) {
```

Comment:
Parameters are not expected to be modified, can be final.

Reply:
compiler will spot that as an optimization; in terms of method behaviour it's not a big deal if the method changes so the variables are non-final so not worth marking IMO